ChatQnA/kubernetes/manifests/README.md

```diff
@@ -14,63 +23,80 @@ You should have all the images
 - llm: opea/llm-tgi:latest
 - chaqna-xeon-backend-server: opea/chatqna:latest
 
+Should you desire to use the Gaudi accelerator, two alternate images are used for the embedding and llm services.
 For Gaudi:
 
 - tei-embedding-service: opea/tei-gaudi:latest
 - tgi-service: ghcr.io/huggingface/tgi-gaudi:1.2.1
 
 > [NOTE]  
-> Please refer README [for Xeon](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/README.md) and [for Gaudi](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/README.md) to build the opea images
+> Please refer README [for Xeon](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/README.md) and [for Gaudi](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/README.md) to build the opea images. These too will be available on Docker Hub soon.
 
-## Deploy Services with Xeon
+## Deploy ChatQnA pipeline
+This involves deploying the ChatQnA custom resource. You can use chatQnA_xeon.yaml or if you have a Gaudi cluster, you could use chatQnA_gaudi.yaml. 
 
-> [NOTE]
-
-- Be sure to modify HUGGINGFACEHUB_API_TOKEN and other important values in qna_configmap_guadi.yaml and qna_configmap_xeon.yaml
-- Be sure the node has path /mnt/models to store all the models
+```sh
+kubectl create ns chatqa
+kubectl apply -f $(pwd)/chatQnA_xeon.yaml
+```
 
-### Deploy
+**GMC will reconcile chatQnA custom resource and get all related components/services ready**
```

Comment:
I think there is a step missed here to create a configmap object, refer to [GMC readme](https://github.com/opea-project/GenAIInfra/tree/main/microservices-connector#to-deploy-on-the-cluster) in GenAIInfra.
```
Create ConfigMap for GMC to hold GenAI Components and GMC Router manifests
kubectl create configmap gmcyaml -n $SYSTEM_NAMESPACE --from-file $(pwd)/config/manifests
```

Reply:
Thank you for the careful review @daisy-ycguo  :-). The config map gets created when the user installs GMC by following the link to instructions. This config map is common to all pipelines and adequate to create once.
Xie xie.